core/rfb.js

```diff
@@ -84,6 +103,10 @@ export default class RFB extends EventTargetMixin {
 
         this._qemuExtKeyEventSupported = false;
 
+        this._clipboardServerCapabilitiesFormats = null;
```

Comment:
If you initiate this containing all the formats as false from the start, you will not have to do all the null checks @CendioOssman talks about below